include/ComputeDivergence.h

```diff
@@ -28,7 +28,7 @@ class ComputeDivergence
   ComputeDivergence();
   ~ComputeDivergence();
 
-  void execute(
+  void operate(
```

Comment:
I don't think this is what @sayerhs meant by changing to an *operator*

Reply:
was he talking about overloading an operator? Not sure what operator would divergence correspond to.